### PR TITLE
Standardize on "hub" in values.yaml

### PR DESCRIPTION
**Pull request by sdake**
Values.yaml has a mix of hub and repository meaning
the same thing.  Use hub in all conditions.

### CODE REVIEW

File: install/kubernetes/helm/istio/README.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Helm charts expose configuration options which are currently in alpha.  The curr
 
 | Parameter | Description | Values | Default |
 | --- | --- | --- | --- |
-| `global.hub` | Specifies the HUB for most images used by Istio | registry/namespace | `docker.io/istio` |
+| `global.repository` | Specifies the HUB for most images used by Istio | registry/namespace | `docker.io/istio` |
```
Comment (costinm) [inline install/kubernetes/helm/istio/README.md]: s/HUB/docker repository/

**Comment by hklai** [comment]
> Why not the other way (default to hub) ? Majority seems to be using hub. What is common usage on other projects ?

+1
It also seems less risky (impact-wise to test/release).

**Comment by sdake** [comment]
hub is fine with me.  I'll adjust PR.

**Comment by sdake** [comment]
starting fresh in a new PR.

**Pull request closed without merging**